### PR TITLE
Remove Dynamic Notch from "Features"

### PR DESCRIPTION
**Pull request by KarateBrot**
# General

This is the counterpart to PR https://github.com/betaflight/betaflight/pull/10673

For **BF 4.3 / API 1.44** onwards I got rid of the "DYNAMIC_FILTER" switch in the configuration tab. Instead I introduced an on/off switch in the filters tab, so finally all filters can be enabled and disabled from inside the filters tab.

Thanks to @haslinghuis for giving me code that was nearly ready to use. He basically made half of this PR.

# Todo

- [x] Wait for #2638 to get merged
- [x] Rebase
- [x] Update dyn notch settings layout according to new rfc slider layout style
- [x] Bugfixes
- [x] Mark PR as ready for review

# Preview

features

dynNotchSwitch

### CODE REVIEW

**Comment by haslinghuis** [comment]
@KarateBrot this needs a rebase now :smiley:

**Comment by ctzsnooze** [comment]
One small comment about the Dynamic Notch switch behaviour.

If Dynamic Notch count is > 0, say '3', and I turn the switch  to 'off' (left position), the switch changes colour to grey, but the value for notch count remains unchanged.  

Usually 'zero' count = 'off', so I would expect the notch count value to go to '0' the moment I move the slider to the left.

It might be more intuitive to do it like that.

The value does go to zero after save, but I guess I am suggesting that it should go to zero before the save.

Otherwise it does what it says :-)

**Comment by ctzsnooze** [comment]
Also, it's a small point, but in other elements, the values are with their labels in the right column, but here the numbers are on the left, squashed in with the switch.

We should be consistent = numbers AND labels to the right (like the rpm filters, immediately above).

Screen Shot 2021-11-26 at 16 31 55
.

**Review by haslinghuis — APPROVED** [review]

**Review by haslinghuis — DISMISSED** [review]
Just tested on 4.2.9 and 4.3

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — DISMISSED** [review]

**Review by klutvott123 — DISMISSED** [review]
Tested. Works 👍

**Comment by KarateBrot** [comment]
I tested it on 4.1, 4.2 and 4.3.

**Review by haslinghuis — APPROVED** [review]

**Review by haslinghuis — DISMISSED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by klutvott123 — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
